### DISCUSSION TITLE
Unsubscribe not working?

### DISCUSSION THREAD

**Discussion by mshelby** [OpenVoiceOS/ovos-buildroot#245] [General]
I love following the project but I'm so busy now that I don't have time. Some time back I must have subscribed to the discussions. Now I need to unsubscribe.

I've tried using the 'unsubscribe' link in the emails, but that only unsubscribes me from that discussion, not all discussions, and any new discussions.

Could an admin here please just unsubscribe me to all threads and future threads in total? Thanks.

**Comment by JarbasAl** [comment]
I'm not sure how to do that, don't think it is exposed to us as admins, it's part of github itself, you should have some button to click somewhere...

![imagem](https://github.com/OpenVoiceOS/ovos-buildroot/assets/33701864/f8c9742d-3c36-42bc-a23f-efcd2a53e288)
